.github/workflows/tests_gpu_kokkos.yml

```diff
@@ -11,6 +11,10 @@ on:
         required: true
         description: The version of PennyLane to use. Valid values are either 'release' (most recent release candidate), 'stable' (most recent git-tag) or 'latest' (most recent commit from master)
   pull_request:
+    paths:
```

Comment:
We also need to run C++ tests for all backends anytime there is a change to a file in
- pennylane_lightning/core/src
- pennylane_lightning/core/src/gates
- pennylane_lightning/core/src/observables
- pennylane_lightning/core/src/measurements
- pennylane_lightning/core/src/utils
- pennylane_lightning/core/src/simulators/base
- pennylane_lightning/core/src/bindings
- pennylane_lightning/core/src/algorithms
and Python tests anytime there is a change in 
- pennylane_lightning/core

Reply:
Ah not a problem, I will update accordingly

Reply:
Done!